src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+    "description": "Shows how many cookies were cleaned and for which domain."
+  },
+
+"contextualIdentitiesEnabledText": {
+    "message": "Aktiviert die Unterstützung für Firefox' \"Contextual Identities\" (EXPERIMENTELL Firefox 53+)",
+    "description": "Enable Support for Firefox's Contextual Identities (EXPERIMENTAL Firefox 53+)"
+  },
+
+"contextualIdentitiesTooltipText": {
+    "message": "NICHT aktivieren solange du nicht Firefox 53+ nutzt. Fügt eine Whitelist für JEDEN Container hinzu, den du nutzt. Die Standard-Whitelist bleibt davon unabhängig.",
+    "description": "Do NOT Enable unless you are on Firefox 53+. This will add a different whitelist for EACH containers you use. Your default whitelist is separate from these."
+  },
+
+"saveText": {
+    "message": "Speichern",
+    "description": "Save"
+  },
+
+"cancelText": {
+    "message": "Abbrechen",
+    "description": "Cancel"
+  },
+
+"defaultSettingsText": {
+    "message": "Standardeinstellungen",
+    "description": "Default Settings"
+  },
+
+"defaultSettingsTooltipText": {
+    "message": "WARNUNG: Diese Aktion wird auch deine URLs zurücksetzen. Erstelle eine Sicherung!",
+    "description": "WARNING: This will also clear your URLS as well. So make a backup of them!"
+  },
+
+"saveConfirmText": {
+    "message": "Gespeichert!",
+    "description": "Saved!"
+  },
+
+"cancelConfirmText": {
+    "message": "Einstellungen wiederhergestellt!",
+    "description": "Restored Settings!"
+  },
+
+"defaultConfirmText": {
+    "message": "Einstellungen auf Standard zurückgesetzt!",
+    "description": "Restored Default Settings!"
+  },
+
+"resetCounterConfirmText": {
+    "message": "Zähler zurückgesetzt!",
+    "description": "Cleared Counter!"
+  },
+
+ "addURLText": {
+    "message": "URL hinzufügen",
+    "description": "Add URL"
+  }, 
+
+"clearURLText": {
+    "message": "Alle löschen",
+    "description": "Remove All"
+  },
+
+"enterURLTooltipText": {
+    "message": "Trage URLs ohne 'www.' ein, drücke dann die Eingabetaste oder klicke auf 'URL hinzufügen'. Alternativ besuche die Seite und füge sie vom blauen Symbol in der Symbolleiste hinzu.",
+    "description": "Enter URLS without the 'www.' part, then press 'Enter' or 'Add URL'. Alternatively go to the site and add it from the blue icon."
+  },
+
+"exportURLSText": {
+    "message": "URLs exportieren",
+    "description": "Export URLS"
+  },
+
+"rightClickToSave": {
+    "message": "Klicke mit der rechten Maustaste auf diesen Link und wähle \"Ziel speichern unter...\"",
+    "description": "Right Click to Save Link As"
+  },
+
+"importURLSText": {
+    "message": "Datei hochladen",
+    "description": "Upload File"
+  },
+
+"reportIssuesText": {
+    "message": "Melde Fehler und schlage neue Funktionen vor",
+    "description": "Report issues and suggest features"
+  },  
+
+"versionText": {
+    "message": "version",
+    "description": "version"
```

Comment:
Now this is tricky… as far as I can see this is used in the about screen for "Chrome version" and "Firefox version". The problem is that you would express this as "Chrome-Version" and "Firefox-Version" in German. Could be changed to a format string.